tomcat-gcp-lib/src/test/java/com/google/cloud/runtimes/tomcat/logging/TraceValveTest.java

```diff
@@ -15,25 +15,24 @@
  */
 package com.google.cloud.runtimes.tomcat.logging;
 
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.anyString;
-import static org.mockito.ArgumentMatchers.contains;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.*;
```

Comment:
Wildcard imports are not permitted by the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports).

@cassand Is this enforced in our checkstyle?

Reply:
The checkstyle does not enforce it at least, but I will change this.